### PR TITLE
feat: add extension named "v2ex" which show hotest ten topics

### PR DESCRIPTION
**Pull request by mopig**
## Description

Add extension named "v2ex" which show hotest ten topics

## Type of change

- New extension named "v2ex"

## Screencast

Screen Shot 2022-01-12 at 14 56 15

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/v2ex/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "title": "v2ex",
+  "description": "hot list of v2ex",
+  "icon": "icon-v2ex.png",
+  "author": "leo",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show v2ex List",
+      "subtitle": "Top 10 hot topics currently",
```
Comment (fe9lix) [inline extensions/v2ex/package.json:12]: I'd leave the subtitle empty and rather update the command title, for example: "Show v2ex Top 10"
Please also see our guidelines for title and subtitle properties: https://developers.raycast.com/basics/prepare-an-extension-for-store

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @mopig, thanks for the contribution! I've left a comment re command title/subtitle. Also, any chance to create a higher-resolution icon? It looks quite blurry...

File: extensions/v2ex/package.json (hunk 1 above)
Comment (mopig) [reply extensions/v2ex/package.json]: I have updated the title and subtitle, thanks.

File: extensions/v2ex/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "v2ex",
+  "title": "v2ex",
+  "description": "hot list of v2ex",
```
Comment (fe9lix) [inline extensions/v2ex/package.json:4]:
```suggestion
  "description": "Read the hot list of v2ex",
```

(The description is displayed in the store, so it should at least be a sentence ;). Feel free to add more if you want.)

**Pull request merged**